src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/ConfigurationClientExtensions.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions
 {
     internal static class ConfigurationClientExtensions
     {
-        public static async Task<KeyValueChange> GetKeyValueChange(this ConfigurationClient client, ConfigurationSetting setting, CancellationToken cancellationToken)
```

Comment:
what changed here?

Reply:
We now expect `IConfigurationClient` instead of `ConfigurationClient`.

Reply:
I think we should leave this class as because it's extension methods of `ConfigurationClient` not `ConfigurationClientCollection`.

You can add `GetKeyValueChange` etc. in the `ConfigurationClientCollection` class just like how you did for other `ConfigurationClient` APIs.

Reply:
These methods are now moved to `FailOverClient` directly.